plugins/kftensorflow/flytekitplugins/kftensorflow/task.py

```diff
@@ -53,9 +53,7 @@ def __init__(self, task_config: TfJob, task_function: Callable, **kwargs):
             task_type=self._TF_JOB_TASK_TYPE,
             task_config=task_config,
             task_function=task_function,
-            requests=task_config.per_replica_requests,
-            limits=task_config.per_replica_limits,
-            **kwargs
+            **{**kwargs, "requests": task_config.per_replica_requests, "limits": task_config.per_replica_limits}
```

Comment:
not sure I understand this.  isn't the result the same?

Reply:
this will override the `requests` and `limits` value in `kwargs` with whatever is in `task_config`. before the diff it would raise a `SyntaxError: keyword argument repeated` error because those two args are being passed explicitly and also unpacked via `kwargs`.

Reply:
error might have cropped up because the `@task` function signature changed recently #401 and now both these args are being forwarded into the task class